kubernetes/services/nginx-storage.yml.j2

```diff
@@ -34,13 +34,13 @@ spec:
             - mountPath:  /etc/nginx/snippets/
               name:       nginx-config-snippets
               readOnly:   true
-            {% if nginx_storage_ssl %}
+            {% if not nginx_storage_allow_http %}
             - mountPath:  /etc/ssl/secrets/
```

Comment:
You should remove all conditions from this file. SSL is now always enabled.

Reply:
Done.